GitOut/Features/Git/Log/GitFileTree.xaml

```diff
@@ -7,14 +7,91 @@
     xmlns:converters="clr-namespace:GitOut.Features.Wpf.Converters"
     xmlns:files="clr-namespace:GitOut.Features.Git.Files"
     xmlns:local="clr-namespace:GitOut.Features.Git.Log"
+    xmlns:app="clr-namespace:GitOut.Features.Wpf.Commands"
     mc:Ignorable="d"
 >
+    <UserControl.Resources>
+        <converters:ToStringConverter x:Key="ToStringConverter" />
+    </UserControl.Resources>
     <Grid>
         <TreeView
             x:Name="FileTree"
             ItemsSource="{Binding RootFiles, RelativeSource={RelativeSource AncestorType={x:Type local:GitFileTree}}}"
             Style="{StaticResource MaterialTreeViewStyle}"
         >
+            <TreeView.ItemContainerStyle>
+                <Style BasedOn="{StaticResource DefaultTreeViewItemStyle}" TargetType="{x:Type TreeViewItem}">
+                    <Setter Property="ContextMenu">
+                        <Setter.Value>
+                            <ContextMenu>
+                                <MenuItem
+                                    Header="{Binding FileName}"
+                                    HeaderStringFormat="Open {0}"
+                                    Command="{x:Static app:Application.Open}"
+                                    CommandParameter="{Binding FullPath}"
+                                >
+                                    <MenuItem.Icon>
+                                        <Path Fill="{DynamicResource MaterialForegroundBase}" Data="{StaticResource OpenInNew}" />
+                                    </MenuItem.Icon>
+                                </MenuItem>
+                                <MenuItem
+                                    Header="Copy filename"
+                                    Command="{x:Static app:Application.Copy}"
+                                    CommandParameter="{Binding FileName, Converter={StaticResource ToStringConverter}}"
```

Comment:
FileName has a property for path right? No need for the string converter in that case.